wdio.conf.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const wdioConf = require('terra-toolkit/lib/wdio/conf');
+const localIP = require('ip');
+const path = require('path');
+
+const staticServerPort = 8080;
+
```

Comment:
I have a few ideas for this file: I have included my changes in [this gist](https://gist.github.com/emilyrohrbough/db688ee72304c69d8be43f9d6de76c28)

This will also keep the package level scripts to `npm run test:wdio ../../wdio.conf.js` and remove the need to 
> create a function that pulls all the package directories and auto-generates suite names...

Reply:
@emilyrohrbough What's the status on this?